roles/satellite-clone/tasks/main.yml

```diff
@@ -60,6 +60,10 @@
   register: enable_repos_result
   when: enable_repos
 
+- name: Enable Ansible 2.9 repository
+  command: subscription-manager repos --enable rhel-{{ ansible_distribution_major_version}}-server-ansible-2.9-rpms
+  when: satellite_version in ["6.9"]
```

Comment:
This can (and should) be done on 6.8 too, we just didn't have the more explicit dependency in the RPM that would trigger the error you posted in 6.8

Reply:
I was testing this as-is and I ran into an error in the middle of execution and I think it had to with the fact that I started the playbook while running ansible 2.4 and ansible was upgraded from underneath: 

```
TASK [satellite-clone : Install packages necessary to check DB status] ***************************************************************************************
Tuesday 08 June 2021  15:28:19 +0000 (0:07:30.279)       0:10:16.888 **********  
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: AttributeError: 'Task' object has no attribute 'async_val'
fatal: [localhost]: FAILED! => {"msg": "Unexpected failure during module execution.", "stdout": ""}

msg: Unexpected failure during module execution.

```
This step completed successfully when I ran it a second time (with ansible 2.9 loaded from the start)

To me it seems that the initial steps in the doc should reflect the correct repo to enable to get the right ansible version installed before the clone process is started

Reply:
I don't understand where this issue comes from, but if using 2.9 fixes it, we should be good.

Reply:
Interestingly, docs don't say to enable Ansible repos: https://access.redhat.com/documentation/en-us/red_hat_satellite/6.9/html/upgrading_and_updating_red_hat_satellite/cloning_satellite_server#sec-Cloning_to_Target